tests/benchmarks/test_encoding.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 
 from ddtrace.ext.ci import CI_APP_TEST_ORIGIN
-from ddtrace.internal.encoding import MsgpackEncoder
+from ddtrace.internal.encoding import Encoder as MsgpackEncoder
```

Comment:
Is this change required for this PR? If not I'd keep the explicit import of `MsgpackEncoder` to avoid breaking tests if the default value of the `Encoder` variable should ever change. We'll then sort this out in the v0.5 PR.